scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterParam.json

```diff
@@ -89,7 +89,8 @@
         "Microsoft-InsightsMetrics",
```

Comment:
we should have tasks to update other templates, bicep and policy, and terraform etc. as well.

Reply:
created